core/tests/protractor/editorAndPlayer.js

```diff
@@ -218,6 +218,8 @@ describe('Full exploration editor', function() {
       general.waitForSystem();
       explorationPlayerPage.clickConfirmRedirectionButton();
       browser.waitForAngular();
+      /* Checking the current url to see if collection_id is preserved in its
+         query string ( It would be the first query field) */
```

Comment:
Format this properly: `... query string (it would be the first query field).`

(no space after '('; end with period. Do this for future comments too.)

Reply:
Done, I had changed it to a one-line comment just after I pushed the previous commit.